### PR TITLE
Add bits required for CoCo support

### PR DESCRIPTION
**Pull request by snir911**
This should used on top of #351 & #348

### CODE REVIEW

File: hack/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+```
+This will create the daemonset in the `openshift-sandboxed-containers-operator` namespace
+and copy the custom shim to `/opt/kata` on all the worker nodes having the label: `node-role.kubernetes.io/kata-oc:`
+
+Create the new MachineConfig to update the Kata configurations
+
+```
+kubectl apply -f mc-coco.yaml
+```
+The MachineConfig will update the CRIO config for the `kata-remote-cc` runtimeClass to point to the custom shim.
```
Comment (bpradipt) [inline hack/README.md:62]: This should be `kata-remote`

File: hack/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+2. Have your Azure's `AZURE_CLIENT_SECRET`, `AZURE_CLIENT_ID` and `AZURE_TENANT_ID` credentials used for the cluster as environment variables
+
+## Setup
+
+Create peer-pods-cm Secret in openshift-sandboxed-containers-operator
+```
+oc create secret generic peer-pods-secret -n openshift-sandboxed-containers-operator --from-literal=AZURE_CLIENT_SECRET=${AZURE_CLIENT_SECRET} --from-literal=AZURE_CLIENT_ID=${AZURE_CLIENT_ID} --from-literal=AZURE_TENANT_ID=${AZURE_TENANT_ID}
+```
+
+Create the peer-pods-cm ConfigMap using the ConfigMap defaulter Job and wait CM to be created
```
Comment (bpradipt) [inline hack/README.md:19]:
```suggestion
Create the `peer-pods-cm` ConfigMap using the ConfigMap defaulter Job and wait for the ConfigMap to be created
```

**Comment by bpradipt** [comment]
@snir911 do you want to also update the instructions with KBS details or we do it as a next PR ?

**Comment by snir911** [comment]
Pointed to your repo, does it make sense?

File: hack/kbs/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+openssl pkey -in kbs.key -pubout -out kbs.pem
+
+# Create a secret object from the kbs.pem file.
+oc create secret generic kbs-auth-public-key --from-file=kbs.pem -n coco-kbs
+```
+
+Create secrets that will be sent to the application by the KBS:
+```
+# Create an application secret
+openssl rand 128 > key.bin
```
Comment (bpradipt) [inline hack/kbs/README.md:30]:
```suggestion
head -c 32 /dev/urandom | openssl enc > key.bin
```

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @snir911

**Pull request merged**